src/wp-includes/block-template-areas.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
Not sure about the naming of this file. Should it be `wp-includes/navigation-areas.php`? Or, alternatively, should this functionality go into the existing `wp-includes/block-template.php?`

Reply:
Yeah this is small enough I'd just put it in `block-template.php`.

Reply:
On second thought I think let's rename to `navigation-areas.php`. I looked at `block-template-utils.php` and `block-template.php` but neither really feel right to me. Looking at `lib/navigation.php` in Gutenberg, there's some more things we probably will want to move over into this file.